### ISSUE TITLE
getToken API 400 bad request

### ISSUE THREAD

**Issue by rfazi** [OpenVidu/openvidu#461]
**Describe the bug**
I'm using NodeJS sdk. When I create the token for the session if for the 'data' field I pass an object instead of a string the server returns 400 bad requests. It was enough for me to do 'JSON.stringify ()' on the value of the 'data' field and I solved it. I advise you internally to convert objects into strings.

**Expected behavior**
The token is created without errors.

**Wrong current behavior**
Server respond 400 bad request code.

**OpenVidu tutorial where to replicate the error**
```
OV.createSession().then(session => {
						session.generateToken({
							data: {test: 'object test'},
							role: OpenViduRole.MODERATOR
						}).then(token => {
							
						}).catch(err => console.error(err));
					}).catch(err => console.error(err));
```

**Comment by pabloFuente** [comment]
Well... The documentation states this very clear, doesn't it? `data` must be a string: https://docs.openvidu.io/en/2.13.0/api/openvidu-node-client/interfaces/tokenoptions.html#data

A different thing is that you think that parameter should also be able to accept JavaScript objects. That's a possible improvement. But the API documentation does not lie.

**Comment by rfazi** [comment]
Yes, the documentation is very clear.
I followed [this](https://docs.openvidu.io/en/2.13.0/reference-docs/openvidu-node-client/) tutorial without opening the documentation first, and it is not indicated here that the parameter must necessarily be a string.
The example only indicates `data: "user_data"`, so I thought it would accept any data until I found out by reading the documentation that only accepts strings.
So yes, mine is some advice for improving the API.
